### PR TITLE
build(android): Bump Android SDK to 5.0.0-beta.1

### PR DESCRIPTION
**Pull request by jennmueng**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring

## :scroll: Description

Bumps `sentry-android` to `5.0.0-beta.1`

## :bulb: Motivation and Context

Bump to latest and prepare for RN preview

## :green_heart: How did you test it?
e2e tests passing

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

**Review by philipphofmann — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - build(ios): Bump sentry-cocoa to 7.0.0 and remove setLogLevel #1459
 - feat: Close method to fully disable the SDK on all layers #1457
+- build(android): Bump Android SDK to 5.0.0-beta.1 #1476
```
Comment (philipphofmann) [inline CHANGELOG.md:7]: One question though: Are there any breaking changes worth mentioning in the Changelog?

Comment (jennmueng) [reply CHANGELOG.md:7]: When I did a run through the changelog, I don't think there were any that would apply to RN. Can you please confirm @marandaneto?

Comment (bruno-garcia) [reply CHANGELOG.md:7]: We agreed nothing that concerns RN so we're good to go here

Comment (marandaneto) [reply CHANGELOG.md:7]: 5.0.0-beta.2 will require changes though

**Pull request merged**